src/actions/download.js

```diff
@@ -64,7 +64,7 @@ async function getDownloadURL({ params }) {
 
   // parse file data
   const provider = this.provider('download', data);
-  const files = types
+  const files = (types && types.length > 0)
```

Comment:
```suggestion
  const files = Array.isArray(types) && types.length > 0
```

could be slightly better, otherwise LGTM

Reply:
I agree. Fixed :)